CDCgov/prime-reportstream
# Cle/7049 fhirpath resolver

This PR adds the ability to define and use 'shorthand' when writing fhirpath filters. It is in the format of %{variable} because StringSubstitutor wouldn't accept the fhir standard %variable (this can be tech debt to address out at a later time if needed).

You will need to reloadTables and reloadSettings to get the new lookup table into your local database.

Test Steps:
1. Run unit tests
OR
1. configure a filter using one or more of the shorthand values defined in the filter_shorthand lookup table
2. Send a message that will hit the universal pipeline
3. verify that it made it through to send, assuming it matches the filters configured.

## Changes
- add filter_shorthand lookup table
- Update organization example to use shorthand
- update fhirRouter default filters to use shorthand
- implement stringSubstitutor in fhirRouter for shorthand string replacement
- update constant resolver to not error out if an item is not found
- update tests

## Checklist

### Testing
